templates/cert-manager.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: kubeaddons.mesosphere.io/v1alpha1
+kind: Addon
+metadata:
+  name: cert-manager
+  namespace: kubeaddons
+  labels:
+    kubeaddons.mesosphere.io/name: cert-manager
+  annotations:
+    appversion.kubeaddons.mesosphere.io/cert-manager: "0.1.0"
+    docs.kubeaddons.mesosphere.io/cert-manager: "https://docs.cert-manager.io/en/release-0.10/"
+spec:
+  kubernetes:
+    minSupportedVersion: v1.7.0
+  cloudProvider:
+    - name: aws
+      enabled: true
+    - name: docker
```

Comment:
Is there anything special in a docker environment for this, or does it `justwork.py`?

Reply:
It `just works (TM)` :)